### PR TITLE
Add ability to disable automatic linking for Github-Style shortcodes

### LINKED ISSUE #1098: [FEATURE] Ability to disable image linking by default
### Describe the feature you want 描述你的功能需求

Hello, this is more of a support question.

Is there a way to use Github-style markdown image `` withut producing an `<a href="image.jpg"></a> container?`
My images are just screenshots and they don't require a link.

I understand this is to a `linked=true` defauilt behavior, but I'm not sure how to disable it so it's disabled for the GH-style markdown.

Thanks!

### Useful reference 有价值的参考

_No response_

### PR DESCRIPTION
**Pull request by DimmKirr**
Fixes https://github.com/HEIGE-PCloud/DoIt/issues/1098

### CODE REVIEW

**Comment by DimmKirr** [comment]
@HEIGE-PCloud can you please review? Thank you!

File: exampleSite/config/_default/params.toml [hunk 1]
```diff
@@ -182,6 +182,9 @@ cacheRemoteImages = false
   # 是否启用阅读时间估计
   enableReadingTime = true
   # Display a message at the beginning of an article to warn the reader that its content might be outdated.
```
Comment (CXwudi) [inline exampleSite/config/_default/params.toml:184]: This line should stay together with line 188 which is the English translation of line 188

File: exampleSite/config/_default/params.toml [hunk 2]
```diff
@@ -182,6 +182,9 @@ cacheRemoteImages = false
...
+  # Whether to enable images to be linked to themselves by default
```
Comment (CXwudi) [inline exampleSite/config/_default/params.toml:185]: Make sure to add a Chinese translation of the English description, you can use any translator like DeepL

File: exampleSite/config/_default/params.toml (hunk 1 above)
Comment (CXwudi) [inline exampleSite/config/_default/params.toml:184]: This comment at line 184 should stay together with the comment at line 188 as it is the English translation of that Chinese description

**Comment by HEIGE-PCloud** [comment]
I believe the link element there is mainly for the light gallery support. The right way to do it might be to disable the links when light gallery is not enabled.

**Comment by HEIGE-PCloud** [comment]
Fixed in https://github.com/HEIGE-PCloud/DoIt/commit/9e7d09945b898ebbea83f9149af91643082a953c

**Pull request closed without merging**